src/diagnostics/Diagnostics.jl

```diff
@@ -30,6 +30,7 @@ import ClimaDiagnostics.Writers: HDF5Writer, NetCDFWriter, DictWriter
 
 import ClimaCore.Operators: column_integral_definite!
 
+using LazyBroadcast: @lazy
```

Comment:
Let's use the `lazy` function instead of the `@lazy` macro. I think it's a bit simpler to reason about.